Seagate/cortx-manager
# EOS-20372: csm_setup test command modifications

Signed-off-by: Udayan Yaragattikar <udayan.yaragattikar@seagate.com>

# Backend

# Problem Statement
https://jts.seagate.com/browse/EOS-20372
- _Overview: JIRA number/GitHub Issue added to PR: [Y/N]:_Y
- _Describe the problem this patch intends to solve._
Integrate csm_gui_test rpm, which will test GUI in headless mode as per provided test plan
## Design

https://seagate-systems.atlassian.net/wiki/spaces/~700883026/pages/416055560/csm-setup+Test+scope

## Coding

- _Coding conventions are followed and code is consistent. Yes/No?_Yes
- _Confirm All CODACY errors are resolved. Yes/No?_Yes

## Testing

- _Confirm that Test Cases are added (for both the cases, fix and feature). Yes/No?_Yes
- _Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability. Yes/No?_Yes
- _Confirm Testing was performed with installed RPM. Yes/No?_Yes
- _Confirm all the ut/regression/smoke test has been performed [Y/N]_Yes

![image](https://user-images.githubusercontent.com/66424882/121819019-81409f00-cca8-11eb-99ee-eb8a0a589675.png)

Validation check for csm-test rpm 
![image](https://user-images.githubusercontent.com/66424882/121819172-5571e900-cca9-11eb-85b6-9ae0b1fee549.png)

![image](https://user-images.githubusercontent.com/66424882/121844330-75cd9200-cd01-11eb-8dec-8958a04c76af.png)

![image](https://user-images.githubusercontent.com/66424882/121845157-a7932880-cd02-11eb-87a8-104103d1eb22.png)

## Checklist

- _PR is self-reviewed? Yes/No?_Yes
- _GitHub Issue is updated_NA
- _Jira is updated_Yes
  -  _Check if the description is clear and explained._ 
    -  _Check Acceptance Criterion is defined._
      -  _All the tests performed should be mentioned before Resolving a JIRA._
        -  _Verification needs to be done before marked as Closed/Verified_ NA
        - _Any interface change Yes/No?_ No
        - _Change notified to other gatekeepers: Yes/No?_NA
        - _Code reviews done and addressed: Yes/No?_Yes
        - _Codacy issues reviewed and addressed: Yes/No?_Yes
        - _Deployment test : Done/Not?_Done
        - _UT/smoke test: Done/Not?_Done
        - _Jira number added to PR: Yes/No?_Yes
        - _Github merge done using UI: Yes/No?_NA
        - _Side effects on other features - deployment/update/node-replacement_ NA
        - _Changes to quick-start-guide/community impact_ NA
        - _All dependent component code PR are raised or already merged Yes/No_ NA
        - _All dependent code already merged in landing branch Yes/No_ NA
